lib/membrane/rtp/vad.ex

```diff
@@ -128,6 +144,11 @@ defmodule Membrane.RTP.VAD do
     end
   end
 
+  @impl true
+  def handle_other({:header_extension_id, id}, _ctx, state) do
+    {:ok, %{state | header_extension_id: id}}
```

Comment:
How about `vad_id` instead of `header_extension_id` in the state?

Reply:
Sounds great!